chart/openfaas/templates/faas-idler-dep.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  app: {{ template "openfaas.name" . }}
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  replicas: {{ .Values.faasIdler.replicas }}
+  template:
+    metadata:
+      labels:
+        app: faas-idler
+    spec:
+      containers:
+        - name: faas-idler
+          image: {{ .Values.faasIdler.image }}
+          imagePullPolicy: {{ .Values.openfaasImagePullPolicy }}
+          env:
+            - name: gateway_url
+              value: {{ .Values.faasIdler.gatewayUrl }}
```

Comment:
Is this URL specified anywhere else too?

Reply:
Unfortunately we don't

Reply:
Unfortunately not

Reply:
I don't want to have to manage an additional string here. This should be derived somehow.